src/module/actor/roll-context/base.ts

```diff
@@ -256,12 +240,12 @@ abstract class RollContext<
             target: unresolved.target?.actor ?? null,
             item,
             domains: this.domains,
-            options: [...this.rollOptions, ...itemOptions, ...markOptions],
+            options: [...this.rollOptions, itemOptions, markOptions, distanceOption ?? []].flat(),
```

Comment:
```suggestion
            options: [this.rollOptions, itemOptions, markOptions, distanceOption ?? []].flat(),
```
You'd do this if you're gonna flatten right?

Reply:
I am flattening right there